### ISSUE TITLE
Tutorial does not run as currently implemented

### ISSUE THREAD

**Issue by DeaglanBartlett** [DeaglanBartlett/roxy#16]
There are a few things that are written in the tutorial which do not match the current code version or lead to errors when run. For example,

- In the section "Markov chain Monte Carlo", the variable `method` is not defined
- In the Gaussian Mixture Model case, the `find_best_gmm` function is called as 
`reg.find_best_gmm(param_names, xobs, yobs, [xerr, yerr], max_ngauss, best_metric='BIC', nwarm=100, nsamp=100, gmm_prior='uniform')` 
but this should be 
`find_best_gmm(params_to_opt, xobs, yobs, xerr, yerr, max_ngauss, best_metric='BIC', infer_intrinsic=True, progress_bar=True, nwarm=100, nsamp=100, gmm_prior='hierarchical', seed=1234, verbose=True)`
- The random seeds should be fixed at every new section to ensure the results are the same for all users

**Comment by DeaglanBartlett** [comment]
Method argument for  section "Markov chain Monte Carlo" fixed in commit 328e2ef0ddafb78b13fa51101f4e030342ea4770.

**Comment by DeaglanBartlett** [comment]
In commit 9edd955d83883dd0d191e43acf6b38ba5084d07c we now fix the GMM tutorial by both correcting the call to `find_best_gmm` and adding random seeds in the correct places.

When fixing this tutorial, I noticed that the `optimise` function was always giving 1.0 for the inferred widths of the Gaussians, which seemed suspicious. This was due to an incorrect assignment of bounds: instead of stating that the weights had to be in the range [0,1], I had made the widths into the range. I have corrected this.
